src/theme.js

```diff
@@ -50,7 +50,7 @@ export const theme = createMuiTheme({
       fontStyle: 'normal',
       fontWeight: 500,
       fontSize: 14,
-      lineHeight: 16,
+      // lineHeight: 16, don't change line height as it's mess up markup
```

Comment:
Which markup? What do you mean? @Batsvinova-Anastasiia , @TomaTomoe please review

Reply:
Your master have visual bug with topbar. I fixed it. Just checkout to master before merge and run app

Reply:
fixed by https://github.com/DarthVanger/NeuralNotes/pull/167/commits/cd171d7bfbb1e3684ed8c396b367f876df031843